cypress/support/pages/common/pageNavigation.js

```diff
@@ -1,12 +1,173 @@
 'use strict'
 
 class Navigation {
-    static #courseOverviewnavigationButton = '[data-testid="Course-Overview"]'
+    static #courseOverviewNavigationButton = '[data-testid="Course-Overview"]'
+    static #teamsOverviewNavigationButton = '[data-testid="Teams"]'
+    static #tasksOverviewNavigationButton = '[data-testid="Aufgaben"]'
+    static #filesOverviewNavigationButton = '[data-testid="Meine Dateien"]'
+    static #personalFilesOverviewNavigationButton = '[data-testid="persönliche Dateien"]'
+    static #coursesFilesOverviewNavigationButton = '[data-testid="Kurse"]'
+    static #teamsFilesOverviewNavigationButton = '[data-testid="Teams"]'
+    static #sharedFilesOverviewNavigationButton = '[data-testid="geteilte Dateien"]'
+    static #newsOverviewNavigationButton = '[data-testid="Neuigkeiten"]'
+    static #calendarOverviewNavigationButton = '[data-testid="Termine"]'
+    static #lernStoreOverviewNavigationButton = '[data-testid="Lern-Store"]'
+    static #addonsOverviewNavigationButton = '[data-testid="Add-Ons"]'
+    static #administrationOverviewNavigationButton = '[data-testid="Verwaltung"]'
+    static #studentAdministrationNavigationButton = '[data-testid="Schüler:innen"]';
+    static #teacherAdministrationNavigationButton = '[data-testid="Lehrkräfte"]';
+    static #courseAdministrationNavigationButton = '[data-testid="Kurse"]';
+    static #classAdministrationNavigationButton = '[data-testid="Klassen"]';
+    static #teamAdministrationNavigationButton = '[data-testid="Teams"]';
+    static #schoolAdministrationNavigationButton = '[data-testid="Schule"]';
```

Comment:
also here semicolon can be removed, like in some variables assignment

Reply:
done